src/main/java/net/snowflake/client/jdbc/SnowflakeType.java

```diff
@@ -450,6 +450,7 @@ public static String javaTypeToClassName(int type) throws SQLException {
         return Double.class.getName();
 
       case Types.TIMESTAMP:
+      case Types.TIMESTAMP_WITH_TIMEZONE:
```

Comment:
Is there a test that needs to be written for this?

Reply:
I added to the test testGetDataTypeWithTimestampTz() in ResultSetLatestIT.java.